.travis.yml

```diff
@@ -1,4 +1,4 @@
-dist: trusty
+dist: xenial
```

Comment:
Xenial build still has an issue: https://github.com/personalrobotics/aikido/issues/363

Reply:
Double-check that this is still an issue in CI, since I believe in the launch scripts I forced it to pull OMPL 1.0 rather than 1.2 to get around this.

Reply:
Looks like it's still failing? https://travis-ci.org/personalrobotics/aikido/jobs/559769715